swagger.json

```diff
@@ -680,37 +680,33 @@
             }
         },
         "/fleet/drivers": {
-            "post": {
-                "tags": [
-                    "Fleet", "Default"
-                ],
-                "summary": "/fleet/drivers",
-                "description": "Get all the drivers for the specified group.",
-                "operationId": "get_fleet_drivers",
+            "get": {
                 "parameters": [
                     {
                         "$ref": "#/parameters/accessTokenParam"
                     },
                     {
-                        "name": "groupDriversParam",
-                        "required": true,
-                        "in": "body",
-                        "schema": {
-                            "type": "object",
-                            "required": [
-                                "groupId"
-                            ],
-                            "properties": {
-                                "groupId": {
-                                    "type": "integer",
-                                    "format": "int64",
-                                    "description": "Group ID to query.",
-                                    "example": 101
-                                }
-                            }
-                        }
+                        "name": "group_id",
+                        "required": false,
+                        "in": "query",
+                        "type": "integer",
+                        "format": "int64",
+                        "description": "Optional Group ID to query for.",
+                        "example": 10566
                     }
                 ],
+                "tags": [
+                    "Fleet", "Default"
+                ],
+                "summary": "/fleet/drivers",
+                "description": "Get all the drivers for the specified group.",
```

Comment:
Let's remove the `group` nomenclature here. Most orgs only have one group and don't know what a group means. Also, can we use the verb `List` here instead of `Get`? 

Maybe `List all the drivers in the fleet.`?

Reply:
sounds good